views/default/resources/user/delete.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$user = get_entity(elgg_extract('guid', $vars));
+if (!$user instanceof \ElggUser) {
+	return;
+}
+
+$content = elgg_view_image_block(elgg_view_entity_icon($user, 'medium'), elgg_view('output/longtext', [
+	'value' => elgg_echo('user:delete:description', [elgg_format_element('strong', [], $user->getDisplayName())]),
+]), ['class' => 'mbl']);
+$content .= elgg_view('core/settings/statistics', [
+	'entity' => $user,
+]);
+
+$content .= elgg_view_form('user/delete', ['action' => 'admin/user/delete'], $vars);
```

Comment:
why pass full `$vars` to the form? Contains way more information than the form would ever need